Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3083,10 +3083,14 @@ void WebPageProxy::executeEditCommand(const String& commandName, const String& a
         return;
     }
 
+    RefPtr focusedFrame = focusedOrMainFrame();
+    if (!focusedFrame)
+        return;
```

Comment:
`callbackFunction` is not called.

Why do we only need to check if frame is available here, but not the other call sites? The other places just pass `std::nullopt` to `grantAccessToCurrentPasteboardData`.